app/src/main/java/org/mozilla/fenix/wallpapers/Wallpaper.kt

```diff
@@ -74,7 +74,29 @@ data class Wallpaper(
          * @param theme One of dark/light.
          * @param name The name of the wallpaper.
          */
-        fun getBaseLocalPath(orientation: String, theme: String, name: String): String =
+        fun legacyGetLocalPath(orientation: String, theme: String, name: String): String =
             "wallpapers/$orientation/$theme/$name.png"
+
+        /**
+         * Defines the standard path at which a wallpaper resource is kept on disk.
+         *
+         * @param type The type of image that should be retrieved.
+         * @param name The name of the wallpaper.
+         */
+        fun getLocalPath(type: ImageType, name: String) = "wallpapers/$name/${type.lowercase()}.png"
```

Comment:
small nit: since name comes first in the path, should it be first in the parameters?

Reply:
I think that's a good idea. Having the string type be first also feels like an improvement to me for some reason lol